go/vt/vttablet/tabletserver/debugenv.go

```diff
@@ -70,90 +70,132 @@ func debugEnvHandler(tsv *TabletServer, w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	switch r.Method {
```

Comment:
this looks like slightly bigger change set than what is needed for a bugfix. are you sure you are not also doing a big of refactoring here?

Reply:
It is a single file change. So, refactor should not cause issues.
Looks like a needed change to make it easy for any future changes.

Reply:
I agree with @systay. Either the refactor should be its own PR, or included in a PR that is adding new dynamic variables. Ideally this PR would have just two changes
- fixing SetTxPoolSize
- changing PoolSize to ReadPoolSize

The second change is very hard to actually see because of the refactor.

Reply:
If you look at the diff as a whole it will be difficult to follow. I have made individual 6 commits to show the changes made with intention.

I did the refactor along because it is done on a single file and does not touch any other component.
If you still feel it is too much for a single PR, I can break it.